daemon/cluster/executor/container/adapter.go

```diff
@@ -133,11 +133,17 @@ func (c *containerAdapter) createNetworks(ctx context.Context) error {
 func (c *containerAdapter) removeNetworks(ctx context.Context) error {
 	for _, nid := range c.container.networks() {
 		if err := c.backend.DeleteManagedNetwork(nid); err != nil {
-			if _, ok := err.(*libnetwork.ActiveEndpointsError); ok {
+			switch err.(type) {
+			case *libnetwork.ActiveEndpointsError:
 				continue
+			case libnetwork.ErrNoSuchNetwork:
+				continue
+			case *libnetwork.UnknownNetworkError:
```

Comment:
Today I clean all the ENV and re-build a binary without UnknownNetworkError, find the issue is also gone.

Great, have submitted the latest change. Please review again. Thanks!